src/App.tsx

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+class App extends React.Component<{}, { count:number }> {
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState((s) => {
+      return { count: s.count + 1 };
+    });
```

Comment:
return is redundant here
```suggestion
    this.setState((s) => ({
      count: s.count + 1
    }));
```